KarhooSDKTests/TestCases/Service/Trips/CancellationFee/CancellationFeeInteractorSpec.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  CancellationFeeInteractorSpec.swift
+//  KarhooSDKTests
+//
+//  Created by Nurseda Balcioglu on 03/12/2020.
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+@testable import KarhooSDK
+
+final class CancellationFeeInteractorSpec: XCTestCase {
+
+    private var mockCancellationFeeRequest: MockRequestSender!
+    private var testObject: KarhooCancellationFeeInteractor!
+
+    private let identifier = "bookingID"
+    
+    private let fee = CancellationFeePrice(currency: "GPB", type: "ABC", value: 10)
+
+    override func setUp() {
+        super.setUp()
+
+        mockCancellationFeeRequest = MockRequestSender()
+        testObject = KarhooCancellationFeeInteractor(requestSender: mockCancellationFeeRequest)
+        testObject.set(identifier: identifier)
+     
+        
+    }
+
+    /**
+     * When: Making a request to cancellation fee
+     * Then: Expected method, path and payload should be set
+     */
+
+    func testRequestFormat() {
+        testObject.execute(callback: { (_:Result<KarhooVoid>) in  })
+
+        let endpoint = APIEndpoint.cancellationFee(identifier: identifier)
+
+        mockCancellationFeeRequest.assertRequestSendAndDecoded(endpoint: endpoint,
+                                                               method: .get,
+                                                               payload: nil)
+    }
+
+    /**
+      * Given: Requesting cancellation fee for a trip
+      * When: The request succeeds with a true
```

Comment:
Perhaps 'When: The request succeeds with a cancellation fee'

Reply:
done